utils/releaser/src/ReleaseWorker/ReleaseCandidate/UpdateChangelogReleaseWorker.php

```diff
@@ -108,4 +88,63 @@ public function getStage(): string
     {
         return Stage::RELEASE_CANDIDATE;
     }
+
+    /**
+     * @return string
+     */
+    private function getSectionsDefinition(): string
```

Comment:
isn't possible to create a params file (https://github.com/github-changelog-generator/github-changelog-generator#params-file) with those configurations? 
I believe it would be less error-prone as you need to copy-paste less text

Reply:
I have moved all possible parameters into file